scripts/utils/formatter.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python3
+# Copyright (c) 2025 The Jaeger Authors.
+# SPDX-License-Identifier: Apache-2.0
+
+import re
+import sys
+
+def extract_section_from_file(file_path, start_marker, end_marker):
+    try:
+        with open(file_path, 'r') as f:
+            text = f.read()
+    except Exception as e:
+        raise Exception(f"error reading file: {e}") from e
```

Comment:
try/except completely unnecessary, without them the result will be the same - program aborted with error message